src/server/setup_resc.c

```diff
@@ -78,11 +78,23 @@ struct resc_sum *svr_resc_sum;
 void
 restart_python_interpreter(const char *caller)
 {
+	int pyrc;
 	log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_HOOK,
 		LOG_INFO, (char *) caller,
 		"Restarting Python interpreter as resourcedef file has changed.");
-	pbs_python_ext_shutdown_interpreter(&svr_interp_data);
-	pbs_python_ext_start_interpreter(&svr_interp_data);
+	pyrc = pbs_python_ext_start_interpreter(&svr_interp_data);
+	if (pyrc != 0) {
+		log_err(PBSE_INTERNAL, (char *) caller, "Failed to restart Python interpreter");
+		/*
+		 * FIXME:
```

Comment:
I think we have to ignore an error here, as callers are IS requests (qmgr resource set/unset/delete/create) and as of they don't return any error.